Tasks/HelmDeployV0/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 166,
+        "Minor": 167,
```

Comment:
There's helm install logic in this task as well. You may want to validate this the logic here for consistency

Reply:
I couldn't find any helm install logic in this task. Also before using this task, the user is supposed to have a HelmInstaller task in the pipeline. Otherwise it's not able to find Helm. Probably that validates the fact that this task does not have any helm install logic?

Reply:
Sounds good.